### ISSUE TITLE
Syntax error in configuration

### ISSUE THREAD

**Issue by bilo-io** [hipstersmoothie/storybook-dark-mode#153]
Hey so, I think there is a syntax error in the `README.md` to set up this addon which would be corrected with the following:

```diff
- export parameters = {
+ export const parameters = {
    // Override the default dark theme
    darkMode: {
```

![image](https://user-images.githubusercontent.com/3089012/117882279-74e0b500-b2aa-11eb-81db-4ebd2477a967.png)

> - I may be wrong in that this is now possible with a later version of ECMAscript, or perhaps something specific to Storybook?

> - However, when I used the code suggestion in the README.md the code would not compile! 
> - This might be an oversight, but let me know in case there's something I'm not aware of ... 
> - Otherwise please update ... for the community
> - I can also create a PR 🤷

**Comment by hipstersmoothie** [comment]
Looks to be! make a pr and I'll merge

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v1.0.9`](https://github.com/hipstersmoothie/storybook-dark-mode/releases/tag/v1.0.9) :rocket:
